module-gui/gui/SwitchData.hpp

```diff
@@ -1,12 +1,15 @@
-// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #pragma once
 
 #include <string>
+#include <optional>
+#include <log/log.hpp>
 
 namespace gui
 {
+    using NameOfAppOptional = std::optional<std::string>;
```

Comment:
Please replace it with the existing type `ApplicationName` if possible. If not, please don't use `Optional` in the name.